### PR TITLE
feat: update code-review-involvement to handle closed but not merged PRs

### PR DESCRIPTION
**Pull request by kathikraemer**

### CODE REVIEW

File: src/metrics/code_review_involvement/index.ts [hunk 1]
```diff
@@ -19,18 +21,21 @@ export const collectCodeReviewInvolvementMetrics = async (
   const repo = payload.repository.name;
   const owner = payload.repository.owner.login;
   const pr_id = payload.pull_request.id;
+  const merged = payload.pull_request.merged;
   const created_at = moment.utc(payload.pull_request.created_at).valueOf();
   const updated_at = moment.utc(payload.pull_request.updated_at).valueOf();
   const closed_at = moment.utc(payload.pull_request.closed_at).valueOf();
   const merged_at = moment.utc(payload.pull_request.merged_at).valueOf();
```
Comment (remmycat) [inline src/metrics/code_review_involvement/index.ts:28]: This should also check for `merged` - currently the output is NaN as seen in the snapshot diff, which is probably not a great fallback. Maybe `null`? That could also be a fallback for the `created_to_merged_duration` and `has_been_merged_by_author fields`

Comment (remmycat) [reply src/metrics/code_review_involvement/index.ts:28]: (The comment is for `merged_at` - with GitHub highlighting only `merged` this seems confusing to me now)

Comment (kathikraemer) [reply src/metrics/code_review_involvement/index.ts]: Good point. I updated the fields that depend on `merged`.

**Review by remmycat — APPROVED** [review]
🙌🏻

**Pull request merged**